src/core/plugins/db-synchronizer.ts

```diff
@@ -60,7 +62,10 @@ export default class DbSynchronizer extends CallablePlugin {
        */
       let permitted = await NetworkIpc.askClusterPermission('db-synchronizer', 20000)
       if (permitted) {
-        this.startMonitoring().catch(e => {});
+        this.nonDeployersSyncLoop().catch(e => log.error(`error when starting monitor %o`, e));
+
+        if(this.nodeManager.currentNodeInfo?.isDeployer)
```

Comment:
This does not handle editing `isDeployer`.
Suppose a node is not a deployer. But we set its `isDeploy` to true later.

`deployersSyncLoop` should run and check `isDeployer` in the function.